portfolio/pom.xml

```diff
@@ -23,11 +23,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
```

Comment:
super nit: along with kofi, i don't think there should be a space between dependencies. here's a style guide for pom files https://maven.apache.org/developers/conventions/code.html

Reply:
Okay sure.